### PR TITLE
fix(uec): certify clean-break closure

### PR DESCRIPTION
**Pull request by jamesryancooper**
Policy reference: `.octon/framework/agency/practices/pull-request-standards.md`
Reminder: Run `@codex review`.

## What

Implements the Unified Execution Constitution clean-break packet on the
dedicated cutover branch and closure-certifies the declared supported
envelope. Normalizes the live workspace-charter path, adds packet runtime
normalization contracts and per-run authority/event evidence, and retains the
two-pass closure bundle plus closure certificate.

## Why

The repo had substantial UEC infrastructure but still mixed packet deltas with
an overclaimed closure state. This change closes the packet issue register,
reconciles the runtime/evidence model to the packet target state, and retains
closure-grade evidence for the supported tuple only.

No-Issue: repository-governance clean-break closure packet

## How

Moved the live workspace machine charter onto `workspace-charter-v1` and
removed compatibility shims from live runtime inputs while preserving them as
historical only. Added packet contracts for run events, event-ledger
manifests, checkpoint v2, state reconstruction, retention, and closure
certificate output. Normalized the certification runs with per-run
`authority/**` roots, `events.ndjson`, `events.manifest.yml`, a supported
Class C external evidence path, and a disclosed non-zero intervention plus
hidden-repair proof. Then froze candidate commit
`079806da0f5668ca2d035a989bb8433facc7ca71`, ran two clean-room certification
passes, and promoted the closure/disclosure surfaces to `complete`.

## Profile Selection Receipt

- Semantic version source(s): `.octon/octon.yml`, `.octon/framework/constitution/charter.yml`
- Release state (`pre-1.0` or `stable`): `pre-1.0`
- `change_profile` (`atomic` or `transitional`): `atomic`
- Hard-gate facts (downtime, coordination, migration/backfill, rollback, blast radius, compliance):
  repo-local constitutional migration; candidate frozen by commit id before
  certification; two detached clean-room worktrees used for Pass 1 and Pass 2;
  no host-native authority widening; closure claim bounded to `MT-B / WT-2 /
  LT-REF / LOC-EN` using `repo-shell` + `repo-local-governed`
- Tie-break status: none
- `transitional_exception_note` (required when `pre-1.0` + `transitional`): n/a

## Implementation Plan

- Workstreams and scope:
  packet objective normalization, authority/runtime/retention contract
  additions, certification-run normalization, closure evidence retention, and
  final claim-discipline promotion
- Public interfaces/types/contracts affected:
  `workspace-charter-v1`, `run-event-v1`, `run-event-ledger-v1`,
  `checkpoint-v2`, `state-reconstruction-v1`,
  `evidence-retention-contract-v1`, `closure-certificate-v1`
- Test scenarios:
  objective-path validation, phase 3 runtime evidence normalization, phase 4
  proof/lab, phase 5 adapters/support targets, phase 7 build-to-delete,
  execution governance, authority exercises, richness validation, disclosure
  claim validation, packet runtime normalization, complete-claim closure
  validation, plus Pass 1 / Pass 2 clean-room reruns
- Assumptions/defaults:
  packet remains non-authoritative under `.octon/inputs/**`; support envelope
  stays bounded to the single supported tuple already declared in
  `support-targets.yml`

## Impact Map (code, tests, docs, contracts)

- Code:
  `.octon/framework/engine/runtime/**`,
  `.octon/framework/orchestration/runtime/_ops/scripts/write-run.sh`,
  `.octon/framework/assurance/governance/_ops/scripts/**`
- Tests/validators:
  `.octon/framework/assurance/runtime/_ops/scripts/**`,
  `.octon/framework/assurance/governance/_ops/scripts/**`
- Docs:
  `.octon/instance/ingress/AGENTS.md`,
  `.octon/state/control/execution/runs/README.md`,
  `.octon/framework/constitution/contracts/{objective,runtime,retention}/README.md`
- Contracts:
  `.octon/framework/constitution/contracts/objective/**`,
  `.octon/framework/constitution/contracts/runtime/**`,
  `.octon/framework/constitution/contracts/retention/**`,
  `.octon/framework/constitution/contracts/disclosure/closure-certificate-v1.schema.json`,
  `.octon/instance/governance/closure/**`

## Compliance Receipt

- [x] Selected exactly one execution profile before planning/implementation.
- [x] Applied release-maturity gate from semantic version.
- [x] Enforced pre-1.0 default (`atomic`) unless hard gates required `transitional`.
- [x] Included `transitional_exception_note` when required.
- [x] Included tie-break escalation when applicable.
- [x] Updated impacted contracts/docs/tests.
- [x] Honored charter change-control constraint (no direct principles charter edits without override evidence).

## Exceptions/Escalations

none

## Tradeoffs

The packet requested a larger runtime rewrite than the repo strictly needed in
every area. Where the existing assurance stack already proved a packet
requirement in substance, this PR closes the issue through explicit evidence
reconciliation instead of churning working subsystems. The retained pass
bundles are verbose, but they keep the closure certificate defensible and
machine-verifiable.

## Testing

Automated:

- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-objective-binding-cutover.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-unified-execution-phase3-runtime-evidence-normalization.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-phase4-proof-lab-enforcement.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-phase5-adapter-support-target-hardening.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-phase7-build-to-delete-institutionalization.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-execution-governance.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-authority-governance-exercises.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-unified-execution-richness.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-assurance-disclosure-expansion.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-support-target-live-claims.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-disclosure-live-roots.sh`
- `bash .octon/framework/assurance/governance/_ops/scripts/validate-uec-packet-runtime-normalization.sh`
- `bash .octon/framework/assurance/governance/_ops/scripts/assert-unified-execution-closure.sh`

Clean-room certification:

- Pass 1 detached worktree at `/tmp/uec-pass1` for commit
  `079806da0f5668ca2d035a989bb8433facc7ca71`
- Pass 2 detached worktree at `/tmp/uec-pass2` for commit
  `079806da0f5668ca2d035a989bb8433facc7ca71`
- Retained pass bundles:
  `.octon/state/evidence/disclosure/releases/2026-04-03-uec-clean-break-closure/closure/pass1/**`
  and
  `.octon/state/evidence/disclosure/releases/2026-04-03-uec-clean-break-closure/closure/pass2/**`

## Rollout

`n/a` for runtime feature flags. This is a clean-break constitutional
migration. The operative rollout control is the frozen candidate commit plus
the retained closure evidence bundle.

## Convivial Purpose Check

- [x] Feature expands genuine user capability (not synthetic engagement).
- [x] Attention and interruption behavior are justified and user-controllable.
- [x] No manipulative patterns or dark-pattern mechanics are introduced.
- [x] Data collection/extraction risk is minimal and explicitly justified.

## Risk Rubric

- Risk class: [ ] Trivial [ ] Low [ ] Medium [x] High
- Rollback plan:
  revert the certification promotion commit first, then revert the runtime
  normalization commit if needed
- Rollback handle:
  git revert of `fix(uec): certify clean-break closure` and
  `refactor(uec): normalize packet closure runtime evidence`
- Flags changed (name, owner, expiry, rollout):
  none
- Autonomy eligibility: [ ] autonomy:auto-merge [x] autonomy:no-automerge

## Contracts and Threat Model

- OpenAPI/JSON-Schema changes:
  `workspace-charter-v1`, `run-event-v1`, `run-event-ledger-v1`,
  `checkpoint-v2`, `evidence-retention-contract-v1`,
  `closure-certificate-v1`
- Threat model update/link:
  bounded by retained authority, intervention, and hidden-repair proof in the
  closure bundle

## Observability and Performance

- Traces/logs/metrics for changed flows:
  run event ledgers, per-run authority bundles, measurement summaries,
  intervention logs, pass logs, and the closure certificate
- Representative traces for high risk changes:
  retained under the pass bundles and normalized run evidence roots
- Performance or bundle impact:
  higher retained evidence volume due to pass bundles and per-run authority /
  event artifacts

## License and Provenance

- New dependencies and licenses:
  none
- Generated code/templates provenance notes:
  none

## Checklist

- [x] Requirements met; edge cases handled
- [x] Security reviewed (authz, input validation, secrets)
- [x] Tests added or updated
- [x] Observability updated (logs, metrics, traces) if needed
- [x] No speculative abstractions or unnecessary complexity
- [x] Conventions followed; no drift introduced
- [x] Non-obvious decisions documented (comments, ADR)
- [x] All review conversations resolved

## Screenshots / Notes

No visual changes. Closure evidence bundle:
`.octon/state/evidence/disclosure/releases/2026-04-03-uec-clean-break-closure/closure/**`

### CODE REVIEW

File: .octon/framework/assurance/governance/_ops/scripts/normalize-uec-packet-certification-runs.sh [hunk 1]
```diff
@@ -0,0 +1,588 @@
...
+  fi
+
+  jq -cn --arg run_id "$run_id" --arg recorded_at "$created_at" --arg decision_ref "$decision_ref" --arg status "$status" '
+    {
+      schema_version: "run-event-v1",
+      event_id: "evt-003-authority-resolution",
+      run_id: $run_id,
+      event_type: (if $status == "failed" or $status == "cancelled" then "authority-denied" else "authority-granted" end),
+      subject_ref: $decision_ref,
+      governing_refs: [$decision_ref, ".octon/state/control/execution/runs/\($run_id)/authority/index.yml"],
```
[the remark this answers is not in the document]
Comment (jamesryancooper) [reply .octon/framework/assurance/governance/_ops/scripts/normalize-uec-packet-certification-runs.sh]: Fixed in `c6e5edee2`. `write_event_ledger` now omits empty decision refs from `governing_refs` and sets `subject_ref` to `null` when no decision artifact exists, so the emitted `run-event-v1` records stay schema-valid for runs like `run-wave4-benchmark-evaluator-20260327`.

File: .octon/framework/assurance/governance/_ops/scripts/normalize-uec-packet-certification-runs.sh [hunk 2]
```diff
@@ -0,0 +1,588 @@
...
+    --arg run_id "$run_id" \
+    --arg ledger_ref ".octon/state/control/execution/runs/$run_id/events.ndjson" \
+    --arg schema_ref ".octon/framework/constitution/contracts/runtime/run-event-v1.schema.json" \
+    --arg updated_at "$updated_at" '
+      {
+        schema_version: "run-event-ledger-v1",
+        run_id: $run_id,
+        ledger_ref: $ledger_ref,
+        event_schema_ref: $schema_ref,
+        event_count: 12,
```
[the remark this answers is not in the document]
Comment (jamesryancooper) [reply .octon/framework/assurance/governance/_ops/scripts/normalize-uec-packet-certification-runs.sh]: Fixed in `c6e5edee2`. The event-ledger manifest now derives `event_count`, `first_event_id`, and `last_event_id` from the events actually emitted into `events.ndjson`, so conditional omission of authority/lease/revocation/checkpoint events no longer leaves the manifest inconsistent.
